EVSL_1.0/TESTS/Gen_MM_Landos/io.c

```diff
@@ -115,8 +115,8 @@ int read_coo_MM(const char *matfile, int idxin, int idxout, cooMat *Acoo) {
   else
     nnz2 = nnz;
   /*-------- Allocate mem for COO */
-  int* IR = (int *)malloc(nnz2*sizeof(int));
-  int* JC = (int *)malloc(nnz2*sizeof(int));
+  int* IR = (int *)calloc(nnz2,sizeof(int));
+  int* JC = (int *)calloc(nnz2,sizeof(int));
```

Comment:
callocs were due to the fact that when I ran a static analysis tool, it detected a route that would result in a reaching lines 161 without having initialized IR,JC with values, but I think that only happens if nnz is 0, at which there would be larger problems.